docs/issue_template.md

```diff
@@ -0,0 +1,22 @@
+# Checklist for author
+
+Remove items that aren't applicable.
+
+## Issue Type
+
+- [ ] Bug Report
+- [ ] Feature Request (requested by stakeholders)
+- [ ] Feature Proposal (possible improvements)
+
+## Issue Description
+
+### Input here issue description
+
+### Extra information
+
+- [ ] Login role (to reproduce, what role should I log in with)
```

Comment:
Mentioning the login role can be important, but I'm not sure if _just_ the role is that useful. Could we reword this into something like

> - [ ] Steps to reproduce the bug or to illustrate the feature request (including login role)

Reply:
I agree, just a role is too specific.